syntax = "proto3";

package envtrackernode;

// The greeting service definition.
service CoreNode {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {}
  rpc SeyGoodbye (GoodbyeRequest) returns (GoodbyeReply) {}
}

// The request message containing the user's name.
message HelloRequest {
  string name = 1;
}

// The response message containing the greetings
message HelloReply {
  string message = 1;
}

// The request message containing the user's name.
message GoodbyeRequest {
  string name = 1;
}

// The response message containing the greetings
message GoodbyeReply {
  string message = 1;
}
